src/Microsoft.TestPlatform.CommunicationUtilities/Resources/Resources.resx

```diff
@@ -126,6 +126,9 @@
   <data name="ConnectionClosed" xml:space="preserve">
     <value>An existing connection was forcibly closed by the remote host.</value>
   </data>
+  <data name="DataCollectorUriForLogMessage" xml:space="preserve">
+    <value>Diagnostic data adapter ('{0}') message: {1}</value>
```

Comment:
Should we not use the known terminology (`Data collector` instead of Diagnostic data adapter)?